### PR TITLE
Feature/iiwa 48

### PR DESCRIPTION
**Pull request by kirbur**

### CODE REVIEW

File: src/backend-api/Startup.cs [hunk 1]
```diff
@@ -76,7 +76,8 @@ public void ConfigureServices(IServiceCollection services)
             services.AddMvc().SetCompatibilityVersion(CompatibilityVersion.Version_2_1);
 
             // Creates a connection to the db in order to make ITInventoryDBContext available to MVC Controllers.
-            services.AddDbContext<ITInventoryDBContext>(options => options.UseSqlServer(Configuration.GetConnectionString("ITInventoryDb")));
+            var connection = @"Server=CQL-INTERN04\SQL16;Database=ITInventoryDB;Trusted_Connection=True;ConnectRetryCount=0";
+            services.AddDbContext<ITInventoryDBContext>(options => options.UseSqlServer(connection));
```
Comment (mtleonard120) [inline src/backend-api/Startup.cs]: This change needs to be reverted - this'll break develop

File: src/frontend-admin/.env [hunk 2]
```diff
@@ -0,0 +1 @@
+REACT_APP_API_URL=https://localhost:44358/api/
```
Comment (mtleonard120) [inline src/frontend-admin/.env]: This file should not be committed to source control - add `.env` to the gitignore

File: src/frontend-admin/src/index.css [hunk 3]
```diff
@@ -1,7 +1,9 @@
...
+body {
+    margin: 0;
+    font-family: cqlRegularFont;
 }
-.body {
-  font-family: cqlBlackFont;
+
+@font-face {
+    font-family: cqlRegularFont;
+    src: url('./content/Fonts/CQL-Regular.otf');
```
Comment (mtleonard120) [inline src/frontend-admin/src/index.css]: Is this correctly importing now?

Comment (kirbur) [reply src/frontend-admin/src/index.css]: yes

File: src/frontend-admin/.env (hunk 2 above)
Comment (mtleonard120) [reply src/frontend-admin/.env]: Bah, you might need to revert the gitignore change, delete this file, commit, then apply the gitignore change again.

File: src/frontend-admin/src/components/App/App.tsx [hunk 4]
```diff
@@ -1,93 +1,71 @@
...
-    </div>
-  );
-};
+                <Switch>
+                    {/* <Route />'s go here */}
+                    <Route path='/dashboard' component={DashboardPage} />
+                    <Route path='/employees' component={EmployeesListPage} />
+                    <Route path='/programs' component={ProgramsListPage} />
+                    <Route path='/hardware' component={HardwareListPage} />
+                    <Route path='/departments' component={DepartmentsListPage} />
```
Comment (mtleonard120) [inline src/frontend-admin/src/components/App/App.tsx]: Would be nice to add a route for `/` in here, like maybe the login page?

Comment (kirbur) [reply src/frontend-admin/src/components/App/App.tsx]: Alex has that on his branch

**Pull request merged**